tests/integ/test_huggingface.py

```diff
@@ -104,3 +106,37 @@ def test_huggingface_training_tf(
         )
 
         hf.fit(train_input)
+
+
+@pytest.mark.release
+@pytest.mark.skipif(
+    integ.test_region() in integ.TRAINING_NO_P2_REGIONS, reason="no ml.p2 instances in this region"
+)
+def test_huggingface_inference(
+    sagemaker_session,
+    gpu_instance_type,
+):
+    env = {
+        "HF_MODEL_ID": "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english",
+        "HF_TASK": "text-classification",
+    }
+    endpoint_name = unique_name_from_base("test-hf-inference")
+
+    model = HuggingFaceModel(
+        sagemaker_session=sagemaker_session,
+        role="SageMakerRole",
+        env=env,
+        image_uri="763104351884.dkr.ecr.us-west-2.amazonaws.com/huggingface-tensorflow-inference:2.4.1-gpu",
```

Comment:
This doesn't need to be hardcoded anymore.

Reply:
right. But i tried on with framework args. Gives me this error:
`The image '763104351884.dkr.ecr.us-east-2.amazonaws.com/huggingface-pytorch-inference:1.7.1-transformers4.6.1-gpu-py36' does not exist..`